openff/bespokefit/tests/schema/test_targets.py

```diff
@@ -134,31 +135,31 @@ def test_check_connectivity_qcfractal_negative(
         ref_data_qcfractal,
         expected_err,
     ):
+        # record is a qcportal.torsiondrive.TorsionDriveRecord
         [(record, _)] = ref_data_qcfractal.to_records()
 
         # Get the first of the final molecules, and prepare an update function so
         # that changes are preserved across calls to get_final_molecule(s)()
-        try:
-            final_molecules = record.get_final_molecules()
-            key, first_final_mol = next(iter(record.cache["final_molecules"].items()))
-
-            def update_record(updated_mol):
-                final_molecules.update({key: updated_mol})
-                record.__dict__["get_final_molecules"] = lambda: final_molecules
-
-        except AttributeError:
-            first_final_mol = record.get_final_molecule()
-
-            def update_record(updated_mol):
-                record.__dict__["get_final_molecule"] = lambda: updated_mol
-
-        # Swap the first two atoms' coordinates to break their connectivity
-        geom = first_final_mol.geometry.copy()
-        geom[0], geom[1] = geom[1], geom[0]
-        updated_mol = first_final_mol.copy(update={"geometry": geom})
 
-        # Update the record with the new geometry
-        update_record(updated_mol)
+        def modify_molecule(molecule):
+            pass
```

Comment:
(not blocking) Could be good to leave a comment explaining this

Reply:
It's not used at all